### PR TITLE
Expose Homeless Service history flag on client dashboard

### PR DESCRIPTION
**Pull request by eanders**
Screen Shot 2020-03-18 at 9 39 49 AM

### CODE REVIEW

File: app/views/clients/_match_results.html.haml [hunk 1]
```diff
@@ -14,7 +14,7 @@
           - sc_count = c.source_clients.count
           - colspan = if sc_count == 1 then 2 else 4 end
           - if sc_count == 1
-            - client_name = "#{client_name} in #{c.source_clients.first.data_source.short_name}"
+            - client_name = "#{client_name} in #{c.source_clients.first.data_source&.short_name}"
```
Comment (awisema) [inline app/views/clients/_match_results.html.haml]: Don't include 'in' if there is no datasource?

File: app/views/clients/_match_results.html.haml [hunk 2]
```diff
@@ -42,7 +42,7 @@
                     .c-checkbox.c-checkbox--round.mr-4
                       = check_box_tag input_id, sc.id, nil, id: sc.id
                       %label{for:sc.id, tabindex:'1' }
-                    %span= "#{sc.full_name} in #{sc.data_source.short_name} <br /> #{sc.uuid}".html_safe
+                    %span= "#{sc.full_name} in #{sc.data_source&.short_name} <br /> #{sc.uuid}".html_safe
```
Comment (awisema) [inline app/views/clients/_match_results.html.haml]: Don't include 'in' if there is no datasource?

File: app/views/clients/_match_results.html.haml (hunk 1 above)
Comment (eanders) [reply app/views/clients/_match_results.html.haml]: We could.  These should never be without a data source in production, but I have some bad data locally, and this allowed me to use the page.

Comment (eanders) [reply app/views/clients/_match_results.html.haml]: Seems like unnecessary work.

**Pull request merged**